src/core_modules/capture-core/components/WidgetIndicator/WidgetIndicatorContent/sortIndicatorsFn.ts

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable complexity */
 import { localeCompareStrings } from '../../../utils/localeCompareStrings';
 
-export const sortIndicatorsFn = (a, b) => {
+/* eslint-disable complexity */
```

Comment:
Revert the move of `eslint` rules to another line. Avoid these types of changes, as they introduce unnecessary differences in the file and complicate the review process. Let's focus solely on refactoring Flow to TypeScript, without improving anything related to `eslint`.